views/_layout.njk

```diff
@@ -70,27 +68,77 @@
             </ul> <!-- /navbar-nav mr-auto -->
           </div> <!-- /navbarCollapse -->
         </div> <!-- /container-fluid-->
-        </nav>
-        {% endblock %} <!-- /end navbar block -->
+      </nav>
+    {% endblock %} <!-- /end navbar block -->
+
+    <!-- container for displaying alert messages to the user -->
+    <div class="container">
+      {% block alerts %}      
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-6 mx-auto">
+              <div id="alertError" class="alert alert-danger collapse">
+                <a href="#" id="closeError" class="close">&times;</a>
+                <strong>Error:</strong> {{message}}
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-lg-6 mx-auto">
+              <div id="alertSuccess" class="alert alert-success collapse">
+                <a href="#" id="closeSuccess" class="close">&times;</a>
+                <strong>Success:</strong> {{message}}
+              </div>
+            </div>
+          </div>
+        </div> <!-- /alerts container -->
+      {% endblock %}
 
     <div class="container">
-      <!-- Add body content from child in this block -->
-      {% block content %}{% endblock %}
+      <!-- Add body from child in this "content" block -->
+      {% block content %}
+      {% endblock %}
     </div> <!-- /container -->
 
     <!-- Footer will contain copyright and/or licence information etc. -->
     {% block footer %}
-        <footer class="footer fixed-bottom navbar-dark bg-dark">
-            <div class="container">
-                <p class="text-muted text-strong">Copyright (c) information and other stuff</p>
-            </div>
-        </footer>
+      <footer class="footer fixed-bottom navbar-dark bg-dark">
+        <div class="container">
+          <p class="text-muted text-strong">Copyright (c) information and other stuff</p>
+        </div>
+      </footer>
     {% endblock %}
     
     {% block scripts %}
-    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js" integrity="sha384-a5N7Y/aK3qNeh15eJKGWxsqtnX/wWdSZSKp+81YjTmS15nvnvxKHuzaWwXHDli+4" crossorigin="anonymous"></script>
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
JQuery's .slim CDN did not come with animation. There is a small overhead for using the bigger library, but from what I read it is more than offset by using the one offered by google. Reason being, LOTS of sites use it so it is likely already cached in the user's browser.